utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -712,6 +712,32 @@ int main(int argc, char **argv)
 
                     CHECK(hipHostFree(lutBuffer));
                 }
+                case 35:
+                {
+                    testCaseName = "glitch";
+                    RpptChannelOffsets *rgbOffsets;
+                    CHECK(hipHostMalloc (&rgbOffsets, batchSize * sizeof(RpptChannelOffsets)));
```

Comment:
Instead of using CHECK Macro
we are using CHECK_RETURN_STATUS macro in the recent changes for the validating the HIP API calls return values
Please use CHECK_RETURN_STATUS here